Lesson7/deniz/index.html

```diff
@@ -1,11 +1,211 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="resp-design.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
+    <title>Medium2</title>
 </head>
 <body>
+    <header>
+
+    </header>
+
+    <main>
+        
+        
+        <div class="container-top">
+
+          <div class=content-one>  
+                <h1 class="h1-top">Time-saving CSS techniques to create responsive images</h1>
+            
+                <div class="profile-info">
+                       <div class="profil-photo">
+                            <img class="small-pic" src="./img/IMG_7036.png" alt="Adrien Zaganelli"> 
+                            </div>
+            
+                            <div class="top-div">
+                            <div class="Adrien-link">
+                            <a href="#">Adrien Zaganelli</a>
+                            <a href="#"><p class="fbtn">Follow</p></a>
+                        </div> 
+                            <div class="date-min">
+                                <span>Published in</span> <a href="#">We've moved to FreecodeCamp.org/news</a><span>3 min read</span><span>Aug 12,2018</span>
+                            </div>  
+               </div>
+            </div>     
+           
+        </div>         
+            
+       
+            <div class="icons">
+                <div class="div-sol">
+                    <a href="#"><i class="fa-solid fa-hands-clapping"><p>10.4K</p></i></a>
+                    <a href="#"><i class="fa-regular fa-comment"><p>21</p></i></a>
+                
+            </div>
+                  <div>
+                    <a href="#"><i class="fa-regular fa-bookmark"></i></a>
+                    <a href="#"><i class="fa-regular fa-circle-play"></i></a>
+                    <a href="#"><i class="fa-solid fa-arrow-up-from-bracket"></i></a>
+                  </div>
+            </div>
+
+
+            <div class="image-container">
+                  <div class="clock-pic">
```

Comment:
Try using <picture> element as described in this guide: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images